lib/service/commands/kill.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Service
+  module Commands
+    module Kill
+      module_function
+
+      TRIGGERS = ["kill"].freeze
```

Comment:
I think term would be a good option too.

Reply:
Unfortunately that is already an active alias for the stop command.